fischkapp/src/components/CreatedCard.tsx/BackCard.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+  StyledCardContent,
+  StyledTextContainer,
+} from "../styles/CardGlobal.styled";
+import { StyledIcon } from "../styles/CardGlobal.styled";
+import editIcon from "../../assets/edit.svg";
+
+type FrontCardPropTypes = {
+  backText: string;
```

Comment:
nitpick: Functions should be typed explicitly, so it's clear what they do. This one should look like:

```
onEdit: () => void;
```

Typing it this way, you know that this function does not accept any arguments, and does not return anything